README.md

```diff
@@ -341,6 +348,8 @@ Property | Type | Default | Description
 **templateFile** | *String* | | path a file containing the `template`
 **data** | *Object* | `{}` | *inherited from [ActionConfig](#interface-actionconfig)*
 **abortOnFail** | *Boolean* | `true` | *inherited from [ActionConfig](#interface-actionconfig)*
+**skip** | *Function* | | *inherited from [ActionConfig](#interface-actionconfig)*
+**transform** | *Boolean* | | an optional function that can be used to transform the file before writing it to disk
```

Comment:
can you move this up above the "inherited" props to follow the pattern of having all inherited props below action specific props?